projects/igniteui-angular/schematics/ng-add/index.ts

```diff
@@ -49,6 +52,25 @@ function LogIncludingDependency(context: SchematicContext, pkg: string, version:
   context.logger.log('info', `Including ${pkg} - Version: ${version}`);
 }
 
+function promptAddPolyfills(tree: Tree, options: Options) {
+  if (options['polyfills']) {
+    const targetFile = 'src/polyfills.ts';
+    if (!tree.exists(targetFile)) {
+      throw new Error(`${targetFile} not found in the src folder.`);
```

Comment:
This could be a warning - you know, on the off chance we can't get to the file, just let the user know the polyfils might need to be tweaked and carry on, rather than erring out the whole schematic